app/vmctl/mimir/mimir.go

```diff
@@ -0,0 +1,235 @@
+package mimir
```

Comment:
This file has copy paste of internal mimir data structures. All structs copied from mimir must have link to the copied source code.